pkg/render/guardian.go

```diff
@@ -81,13 +81,14 @@ func GuardianPolicy(cfg *GuardianConfiguration) (Component, error) {
 
 // GuardianConfiguration contains all the config information needed to render the component.
 type GuardianConfiguration struct {
-	URL               string
-	PullSecrets       []*corev1.Secret
-	Openshift         bool
-	Installation      *operatorv1.InstallationSpec
-	TunnelSecret      *corev1.Secret
-	TrustedCertBundle certificatemanagement.TrustedBundle
-	TunnelCAType      operatorv1.CAType
+	URL                 string
+	PullSecrets         []*corev1.Secret
+	Openshift           bool
+	Installation        *operatorv1.InstallationSpec
+	TunnelSecret        *corev1.Secret
+	TrustedCertBundle   certificatemanagement.TrustedBundle
+	TunnelCAType        operatorv1.CAType
+	EnterpriseCRDExists bool
```

Comment:
Instead of using the presence of the enterprise CRD, we should use `c.cfg.Installation.Variant`, which is already available in the config.